translations/en.json

```diff
@@ -4,5 +4,62 @@
   "Medications - current": "Medications - current",
   "Medications - past": "Medications - past",
   "Since": "Since",
-  "care programs": "Care Programs"
+  "care programs": "Care Programs",
+  "Immunizations": "Immunizations",
+  "vaccine": "VACCINE",
+  "recent vaccination": "RECENT VACCINATION",
```

Comment:
Things like only uppercase should be enforced via styling rules - not the translation phrases.

Reply:
@FlorianRappl, This is actually the translation we are putting for the vaccine and recent vaccinations. However, we will make sure to use `uppercase` at the places this text is used.

Reply:
Yes sure, but the translation is rather "recent vaccination", right @mddubey ? If you want to make it uppercase then this is only for design purposes - which is why CSS exist.